src/NBitcoin/BIP32/ExtPubKey.cs

```diff
@@ -72,11 +72,11 @@ internal ExtPubKey()
         {
         }
 
-        public ExtPubKey(byte[] bytes)
+        public ExtPubKey(Network network, byte[] bytes)
```

Comment:
I think we can remove this from the constructor `ExtPubKey` is not one of the types in consensus factory.